src/arrayMethodReduce.js

```diff
@@ -5,7 +5,16 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let prev
+      = (arguments.length < 2) ? this[0] : initialValue;
+    const start
+      = (arguments.length < 2) ? 1 : 0;
```

Comment:
не делай так перенос, это некорректно
```suggestion
    let prev = (arguments.length < 2) ? this[0] : initialValue;
    const start = (arguments.length < 2) ? 1 : 0;
```